imagetool/src/main/resources/docker-files/Create_Image.mustache

```diff
@@ -65,17 +65,22 @@ RUN tar xzf {{{tempDir}}}/$JAVA_PKG -C /u01 \
 FROM OS_UPDATE as WLS_BUILD
 LABEL com.oracle.weblogic.imagetool.buildid="{{buildId}}"
 ARG WLS_PKG
-ARG INV_LOC
 ARG ORAINST
 
 ENV WLS_PKG=${WLS_PKG:-fmw_12.2.1.3.0_wls_Disk1_1of1.zip} \
     JAVA_HOME={{{java_home}}} \
     ORACLE_HOME={{{oracle_home}}} \
-    INV_LOC=${INV_LOC:-/u01/oracle/oraInventory} \
+    INV_LOC={{inv_loc}} \
```

Comment:
Why do we need this environment variable?  Looks like it could be removed.

Reply:
removed